### PR TITLE
Update references to "kubectl run" in docs

### PR DESCRIPTION
**Pull request by janetkuo**
To address the user complaints about "can't find rc with `kubectl run`". 

@kubernetes/docs @pwittrock

### CODE REVIEW

**Comment by janetkuo** [comment]
Published in http://janetkuo.github.io/docs/:
- http://janetkuo.github.io/docs/admin/limitrange/
- http://janetkuo.github.io/docs/admin/namespaces/
- http://janetkuo.github.io/docs/admin/resourcequota/
- http://janetkuo.github.io/docs/user-guide/config-best-practices/
- http://janetkuo.github.io/docs/user-guide/debugging-services/
- http://janetkuo.github.io/docs/user-guide/docker-cli-to-kubectl/
- http://janetkuo.github.io/docs/user-guide/pods/single-container/

File: docs/user-guide/config-best-practices.md [hunk 1]
```diff
@@ -108,6 +108,6 @@ This document is meant to highlight and consolidate in one place configuration b
 
 - Use kubectl bulk operations (via files and/or labels) for get and delete. See [label selectors](/docs/user-guide/labels/#label-selectors) and [using labels effectively](/docs/user-guide/managing-deployments/#using-labels-effectively).
 
-- Use `kubectl run` and `expose` to quickly create and expose single container replication controllers. See the [quick start guide](/docs/user-guide/quick-start/) for an example.
+- Use `kubectl run` and `expose` to quickly create and expose single container Deployments. See the [quick start guide](/docs/user-guide/quick-start/) for an example.
```
Comment (pwittrock) [inline docs/user-guide/config-best-practices.md]: Later this file says deployments are not enabled by default.  Is this still true?  Maybe we should also delete the section about rolling updates with replication controllers

File: docs/user-guide/debugging-services.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 An issue that comes up rather frequently for new installations of Kubernetes is
 that `Services` are not working properly.  You've run all your `Pod`s and
-`ReplicationController`s, but you get no response when you try to access them.
+`Deployment`s, but you get no response when you try to access them.
```
Comment (pwittrock) [inline docs/user-guide/debugging-services.md]: This document stays k8s doesn't support interactive pods.  Would you update it while you are here?

File: docs/user-guide/config-best-practices.md (hunk 1 above)
Comment (pwittrock) [inline docs/user-guide/config-best-practices.md]: This document says the service should be created before the RC / RS.  Is this still true?  The majority of our examples create the RC/RS first.  We should be consistent in how we tell users to do things.

Comment (bgrant0607) [inline docs/user-guide/config-best-practices.md]: Deployment is enabled by default.

Yes, the reasons to start services first still apply.

**Comment by pwittrock** [comment]
Review pass finished

File: docs/user-guide/config-best-practices.md (hunk 1 above)
Comment (bgrant0607) [inline docs/user-guide/config-best-practices.md]: Note that starting services first is one of these issues about teaching vs. production-ready solutions, door before key, etc.

Comment (janetkuo) [inline docs/user-guide/config-best-practices.md]: Removed the "deployment is not enabled by default" part. Will do the rest in a follow-up PR.

**Comment by janetkuo** [comment]
Removed the "deployment is not enabled by default" part and will address the rest in a follow-up PR. I want to get this in ASAP since this doc/behavior inconsistency confuses lots of users.

File: docs/admin/namespaces/index.md [hunk 3]
```diff
@@ -38,12 +38,12 @@ For this exercise, we will create two additional Kubernetes namespaces to hold o
...
 Let's imagine a scenario where an organization is using a shared Kubernetes cluster for development and production use cases.
 
-The development team would like to maintain a space in the cluster where they can get a view on the list of pods, services, and replication controllers
+The development team would like to maintain a space in the cluster where they can get a view on the list of Pods, Services, and Deployments
 they use to build and run their application.  In this space, Kubernetes resources come and go, and the restrictions on who can or cannot modify resources
 are relaxed to enable agile development.
 
 The operations team would like to maintain a space in the cluster where they can enforce strict procedures on who can or cannot manipulate the set of
-pods, services, and replication controllers that run the production site.
+pods, services, and Deployments that run the production site.
```
Comment (bgrant0607) [inline docs/admin/namespaces/index.md]: Nit (can be fixed in followup): Deployments has different capitalization than pods and services.

**Comment by bgrant0607** [comment]
Merging. 

In a follow-up, we should explain that kubectl run and other commands return the type and name of the resource created/mutated, which can then be used in subsequent commands. We could also link to docs in the 1.1 branch and explain how to use --generator=run/v1 to get 1.1 behavior.

**Pull request merged**